modules/internal/BytesCasts.chpl

```diff
@@ -268,14 +268,27 @@ module BytesCasts {
     return ret;
   }
 
+  //
+  // casts from bytes to c_ptrConst(c_char/int(8)/uint(8))
+  //
+  inline operator :(x: bytes, type t:c_ptrConst(?eltType))
```

Comment:
I would not expect that we want to allow this cast. I would expect that we need people wanting to do this to call `c_ptrTo(myBytes)`.

Reply:
removed